tfhe/examples/utilities/params_to_file.rs

```diff
@@ -98,9 +138,22 @@ impl ParamDetails<u64> for CompactPublicKeyEncryptionParameters {
         panic!("polynomial_size not applicable for compact public-key encryption parameters")
     }
 
-    fn log_ciphertext_modulus(&self) -> usize {
-        assert!(self.ciphertext_modulus.is_native_modulus());
-        64
+    fn lwe_ciphertext_modulus(&self) -> ParamModulus {
+        let ct_modulus = self.ciphertext_modulus;
+        if ct_modulus.is_native_modulus() {
+            ParamModulus::Other(1u128 << ct_modulus.associated_scalar_bits())
+        } else {
+            ParamModulus::Other(ct_modulus.get_custom_modulus())
+        }
+    }
+
+    fn glwe_ciphertext_modulus(&self) -> ParamModulus {
+        let ct_modulus = self.ciphertext_modulus;
+        if ct_modulus.is_native_modulus() {
+            ParamModulus::Other(1u128 << ct_modulus.associated_scalar_bits())
+        } else {
+            ParamModulus::Other(ct_modulus.get_custom_modulus())
+        }
```

Comment:
I think we can panic!/unimplemented! here as the compact pk parameters are used as LWE parameters